pronotepy/ent.py

```diff
@@ -11,20 +11,25 @@
 log = getLogger(__name__)
 log.setLevel(DEBUG)
 
+HEADERS = {
+    'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:73.0) Gecko/20100101 Firefox/73.0'
+}
 
-@typing.no_type_check
-def educonnect(url: str, session, username: str, password: str):
-    headers = {
-        'connection': 'keep-alive',
-        'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:73.0) Gecko/20100101 Firefox/73.0'}
+PAYLOAD = {
+    'email': username,
+    'password': password,
+}
```

Comment:
Uhhh... This works? Isn't `username` and `password` undefined?

Reply:
Oh yeah true. I'm so dumb. I will see to do it differently.

Reply:
I think it's not a very good idea to use constant for payload😅

Reply:
Yeah 😅. No worries I already found a solution.